translations/ui-orders/en.json

```diff
@@ -193,10 +193,80 @@
+  "exportSettings.fields.approved": "Approved",
+  "exportSettings.fields.poLineNumber": "POLine number",
+  "exportSettings.fields.titleOrPackage": "Title",
+  "exportSettings.fields.subscriptionFrom": "Subscription from",
+  "exportSettings.fields.subscriptionTo": "Subscription to",
+  "exportSettings.fields.subscriptionInterval": "Subscription interval",
+  "exportSettings.fields.receivingNote": "Receiving note",
+  "exportSettings.fields.publisher": "Publisher",
+  "exportSettings.fields.edition": "Edition",
+  "exportSettings.fields.packagePoLineId": "Linked package",
+  "exportSettings.fields.contributor": "Contributors",
+  "exportSettings.fields.productIdentifier": "Product identifiers",
+  "exportSettings.fields.acquisitionMethod": "Acquisition method",
+  "exportSettings.fields.orderFormat": "Order format",
+  "exportSettings.fields.createdDateLine": "Created on",
+  "exportSettings.fields.receiptDate": "Receipt date",
+  "exportSettings.fields.receiptStatus": "Receipt status",
+  "exportSettings.fields.paymentStatus": "Payment status",
+  "exportSettings.fields.source": "Source",
+  "exportSettings.fields.donor": "Donor",
+  "exportSettings.fields.selector": "Selector",
+  "exportSettings.fields.requester": "Requester",
+  "exportSettings.fields.cancellationRestriction": "Cancellation restriction",
+  "exportSettings.fields.cancellationRestrictionNote": "Cancellation description",
+  "exportSettings.fields.rush": "Rush",
+  "exportSettings.fields.collection": "Collection",
+  "exportSettings.fields.poLineDescription": "Line description",
+  "exportSettings.fields.instructions": "Instrucitons to vendor",
+  "exportSettings.fields.vendorAccount": "Account number",
+  "exportSettings.fields.listUnitPrice": "Physical unit price",
+  "exportSettings.fields.quantityPhysical": "Quantity physical",
+  "exportSettings.fields.listUnitPriceElectronic": "Electronic unit price",
+  "exportSettings.fields.quantityElectronic": "Quantity electronic",
+  "exportSettings.fields.discount": "Discount",
+  "exportSettings.fields.poLineEstimatedPrice": "Estimated price",
+  "exportSettings.fields.fundDistribution": "Fund distribution",
+  "exportSettings.fields.location": "Location",
+  "exportSettings.fields.materialSupplier": "Material supplier",
+  "exportSettings.fields.receiptDue": "Receipt due",
+  "exportSettings.fields.expectedReceiptDate": "Expected receipt date",
+  "exportSettings.fields.volumes": "Volumes",
+  "exportSettings.fields.createInventory": "Create inventory",
+  "exportSettings.fields.materialType": "Material type",
+  "exportSettings.fields.accessProvider": "Access provider",
+  "exportSettings.fields.activated": "Activation status",
+  "exportSettings.fields.activationDue": "Activation due",
+  "exportSettings.fields.createInventoryE": "Create inventory E",
+  "exportSettings.fields.materialTypeE": "Material type E",
+  "exportSettings.fields.trial": "Trial",
+  "exportSettings.fields.expectedActivation": "Expected activation",
```

Comment:
IIRC, we talked with Dennis regarding translations. Since it will be hard to support with back-end solution, it's better to not introduce them as well with FE approach